js/mainjs/initWindow.js

```diff
@@ -49,13 +48,12 @@ module.exports = function(config) {
 
 	// Load the index.html of the app.
 	mainWindow.loadURL('file://' + __dirname + '/../../index.html');
-
 	// Choose not to show the menubar
 	if (process.platform !== 'darwin') {
 		mainWindow.setMenuBarVisibility(false);
 	} else {
 		// Create the Application's main menu - OSX version might feel weird without a menubar
-		Menu.setApplicationMenu(AppMenu);
+		Menu.setApplicationMenu(new AppMenu(mainWindow));
```

Comment:
I still haven't fully grasped the `new` operator. What purpose does it serve here?